src/main/java/core/basesyntax/Output.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Output {
+    String printAllInformation();
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. However, in this case, since the interface only contains one method related to outputting information, it's fine. Just make sure that any additional methods you might add in the future adhere to the single responsibility principle.